### PR TITLE
HW2: Rebase HW1 onto main branch

### PR DESCRIPTION
**Pull request by kurttepelikerim**

### CODE REVIEW

File: projects/python/image_augmentation.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# image-augmentation
+
+[![](https://img.shields.io/badge/project-link-green)](https://github.com/kurttepelikerim/image_augmentor)
```
Comment (timkpaine) [inline projects/python/image_augmentation.md:3]: Bad link

File: students/kk3084.md [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
+|:--|:--|
+|Name|Kerim Kurttepeli|
+|UNI| kk3084|
+| GitHub Profile | [link](https://github.com/kurttepelikerim) |
+| GitHub README | [link](https://github.com/kurttepelikerim/kurttepelikerim/blob/main/README.md) |
 
 ## Project Proposal
-
-One possible idea I have for the project is to create library of predefined image augmentation presets. This means that given some training images the library artificially creates more training images through different ways of processing such as random rotation, shifts, shear, and flips, etc. A simple interface for image augmentations would be ideal.
+- [image-augmentation](./projects/python/image_augmentation.md)
```
Comment (timkpaine) [inline students/kk3084.md:11]: update link path and re-squash 😈

File: students/kk3084.md [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
 
 ## Project Contributions
-
-There are some good first issues for hugginface's transfomer library which I want to contribute to. I have used huggingface once or twice before and would like to see if I can contribute. Other libraries which I have used extensively are Tenserflow and Pytorch. Another option is to contribute to these libraries which also have some good first issues.
+- [huggingface](./projects/python/huggingface.md)
```
Comment (timkpaine) [inline students/kk3084.md:14]: update link path and re-squash 😈

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: projects/python/image_augmentation.md (hunk 1 above)
Comment (kurttepelikerim) [reply projects/python/image_augmentation.md]: I mistakenly linked the wrong project path but the original project repo was created before the hw2 deadline ('Updated 4 days ago').

**Pull request merged**